README.md

```diff
@@ -12,8 +12,7 @@ e.g. `../ansible/playbooks/playbook.yml`
 ```
 ---
 - hosts: localhost
-  vars:
-    gather_facts: true
+  gather_facts: True
   tasks:
```

Comment:
We should use `check_mode: no` (modern) or `always_run: yes` (deprecated) option in order to fill virtual inventory with host in `dry_run` mode.